lib/iron_bank/resources/rate_plan_charge.rb

```diff
@@ -6,7 +6,11 @@ module Resources
     #
     class RatePlanCharge < Resource
       def self.excluded_fields
-        super + %w[RolloverBalance]
+        super + separate_query_fields
+      end
+
+      def self.separate_query_fields
```

Comment:
WDYTA moving to a private constant (eg., `SINGLE_RESOURCE_QUERY_FIELDS`)?

Reply:
Good point for the naming, but won't be using a constant in just that file since it would require introspection to check if the class defines it.